angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -219,12 +231,18 @@ import {
         </div>
 
         <span
-          class="sprk-c-Divider sprk-u-mvn sprk-u-mhm"
+          sprkDivider
```

Comment:
I enhanced the footer component to use the sprkDivider directive while i was in here, also added a footer specific divider class.